Assets/Scripts/UI/Menus/SettingsMenu.cs

```diff
@@ -35,13 +35,23 @@ private void Awake()
         /// </summary>
         public void LoadSettings()
         {
+            if (Application.platform == RuntimePlatform.Android || 
+                Application.platform == RuntimePlatform.IPhonePlayer)
+            {
+                SetHighestQuality();
+                PlayerPrefs.DeleteAll();
+                resolutionsDropdown.gameObject.SetActive(false);
+            }
```

Comment:
Kan je hier ergens ook landscape mode kunnen aanroepen? Als dat er in zit dan zou een nieuwe functie met de naam `LoadMobileSettings` hier goed passen.